shared/video-sdk/get-started/get-started-sdk/project-implementation/unreal.mdx

```diff
@@ -61,19 +61,31 @@ To implement this interface, do the following:
         * **Size X** - 130
         * **Size Y** - 60
 
-    1. Use the same procedure to create a button called `LeaveBtn` and change the following properties in **Details**.
+    1. Use the same procedure to create a button called `LeaveBtn` and change the following properties in **Details**:
 
         * **Position X** - 1150
         * **Position Y** - 960
         * **Size X** - 130
         * **Size Y** - 60
 
-    1. Add a label to `JoinBtn`.
+    1. Add a label to `JoinBtn`:
 
-        1. Drag **Text** from the **Common** section of **Palette Panel** to **Hierarchy** and drop it over `JoinBtn`.
-        1. In **Details**, change the **Text** field to `Join`.
+        1. Drag **Text** from the **Common** section of **Palette Panel** to the canvas panel.
+        1. Change the following properties in **Details**:
 
-    1. Use the same procedure and add a text widget for the leave button where the **Text** field says `Leave`. 
+            * **Position X** - 1030
+            * **Position Y** - 972
+            * **Size X** - -15
+            * **Size Y** - -15
+            * **Text** - `Join`
+
+    1. Use the same procedure to update `LeaveBtn` with the following properties:
+
+        * **Position X** - 1168
+        * **Position Y** - 972
+        * **Size X** - -15
+        * **Size Y** - -15
+        * **Text** - `Leave`
```

Comment:
Yes I test it . This is correct what you said. But I made these changes because of one reason. What I noticed that when I put cursor over the Text button(wich is over the join or leave button), the button didn't focus. While it is expected that as I put the cursor any where in the join or leave button, the complete button should be focussed which shows that it can be pressed. It is only focussed when I put my cursor at the very end boundaries which is out of Text button boundary (very less area , as Text button area covers almost all leave ot join button area). That's why I made text button 's Size x and Size Y almost infinitely small (-15) , Now Entire Leave or Join Button is available for cursor to be focussed correctly.